src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -190,17 +192,15 @@ func parseFlags() {
 	}
 	// Note - we can ignore the returned error because CommandLine.Parse() will
 	// exit if it fails.
-	flag.CommandLine.Parse(args)
+	_ = flag.CommandLine.Parse(args)
```

Comment:
Same as above. If we're ignoring the error, why do we need the underscore at all? Just to be explicit we're ignoring it?